docs/en/developers/01_First_steps.md

```diff
@@ -1,6 +1,47 @@
-# Environment configuration
+# Environment configuration (Docker)
 
-**TODO**
+FreshRSS is built with PHP and use a homemade framework, Minz. The dependencies are directly included in the source code and so you don't need Composer.
+
+There are various ways to configure your development environment, the easiest and most supported one is based on Docker. This is the solution documented below. If you already have a working PHP environment, you probably don't need it.
+
+We assume here that you use a GNU/Linux distribution, capable to run Docker. Otherwise, you'll have to adapt the commands to your case.
+
+The commands that follow have to be executed in a console. They start by `$` when commands need to be executed as normal user, and by `#` when they need to be executed as root user. You don't have to type these characters. A path may be indicated before to help you to identify where they need to be executed. For instance, `app$ echo 'Hello World'` indicates that you have to execute `echo` command under `app/` directory.
+
+First, you need to install [Docker](https://docs.docker.com/install/linux/docker-ce/ubuntu/).
+
+Once you're done, clone the repository with:
+
+```console
+$ git clone git@github.com:FreshRSS/FreshRSS.git
+$ # or
+$ git clone https://github.com/FreshRSS/FreshRSS.git
+$ cd FreshRSS
+```
+
+Note that, if you want to contribute, you have to fork the repository first and clone your fork instead of the "root" one. Adapt the commands in consequence.
+
+Then, the only command you need to know is the following:
+
+```console
+$ make start
+$ # or if your user is not in the `docker` group
+$ SUDO=sudo make start
```

Comment:
Is there a particular reason not just to use `sudo make start`?

Reply:
It's because in the previous version, I used the current user UID and GID to set on the PHP containers. By calling `sudo` directly, the values weren't the correct ones:

```console
[marien@pizza FreshRSS]$ id
uid=1000(marien) gid=1000(marien) groupes=1000(marien),10(wheel)
[marien@pizza FreshRSS]$ sudo id
uid=0(root) gid=0(root) groupes=0(root)
```

Now it's not required anymore indeed, but I do not like the current behavior which change the ownership of the files in the `Docker/entrypoint.sh` file. I don't have the courage to change that though: it works correctly like that and there are more important things to work on :).